meetupselector/user/models.py

```diff
@@ -27,6 +27,16 @@ def create_user(self, email, password=None, **extra_fields):
         extra_fields.setdefault("is_superuser", False)
         return self._create_user(email, password, **extra_fields)
 
+    def create_staffuser(self, email, password, **extra_fields):
+        """Create a superuser."""
+        extra_fields.setdefault("is_superuser", False)
+        extra_fields.setdefault("is_staff", True)
+
+        if extra_fields.get("is_staff") is not True:
+            raise ValueError("Staffuser must have is_staff=True.")
+
+        return self._create_user(email, password, **extra_fields)
+
```

Comment:
I have doubts if this method is necessary.

Reply:
You can use User.objects.create_user(email, password, is_staff=True)

Reply:
As above: I had problems creating a staff user using the User.objects.create_user(email, password, is_staff=True) method. This was a solution to create a staff member and not a supersuser.